diff --git a/node_modules/gatsby-plugin-sharp/index.js b/node_modules/gatsby-plugin-sharp/index.js
index c98f03b..9bf950b 100644
--- a/node_modules/gatsby-plugin-sharp/index.js
+++ b/node_modules/gatsby-plugin-sharp/index.js
@@ -512,9 +512,14 @@ async function fluid({
   ]; // use standard breakpoints if no custom breakpoints are specified
 
   if (!options.srcSetBreakpoints || !options.srcSetBreakpoints.length) {
-    fluidSizes.push(options[fixedDimension] / 4);
-    fluidSizes.push(options[fixedDimension] / 2);
-    fluidSizes.push(options[fixedDimension] * 1.5);
+    fluidSizes.push(options[fixedDimension] * 0.2);
+    fluidSizes.push(options[fixedDimension] * 0.4);
+    fluidSizes.push(options[fixedDimension] * 0.6);
+    fluidSizes.push(options[fixedDimension] * 0.8);
+    fluidSizes.push(options[fixedDimension] * 1.2);
+    fluidSizes.push(options[fixedDimension] * 1.4);
+    fluidSizes.push(options[fixedDimension] * 1.6);
+    fluidSizes.push(options[fixedDimension] * 1.8);
     fluidSizes.push(options[fixedDimension] * 2);
   } else {
     options.srcSetBreakpoints.forEach(breakpoint => {
