diff --git a/node_modules/gatsby-plugin-sharp/index.js b/node_modules/gatsby-plugin-sharp/index.js
index ea4fd6b..b4382a2 100644
--- a/node_modules/gatsby-plugin-sharp/index.js
+++ b/node_modules/gatsby-plugin-sharp/index.js
@@ -518,10 +518,17 @@ async function fluid({
   ]; // use standard breakpoints if no custom breakpoints are specified
 
   if (!options.srcSetBreakpoints || !options.srcSetBreakpoints.length) {
-    fluidSizes.push(options[fixedDimension] / 4);
-    fluidSizes.push(options[fixedDimension] / 2);
-    fluidSizes.push(options[fixedDimension] * 1.5);
+    fluidSizes.push(options[fixedDimension] * 0.2);
+    fluidSizes.push(options[fixedDimension] * 0.4);
+    fluidSizes.push(options[fixedDimension] * 0.6);
+    fluidSizes.push(options[fixedDimension] * 0.8);
+    fluidSizes.push(options[fixedDimension] * 1.2);
+    fluidSizes.push(options[fixedDimension] * 1.4);
+    fluidSizes.push(options[fixedDimension] * 1.6);
+    fluidSizes.push(options[fixedDimension] * 1.8);
     fluidSizes.push(options[fixedDimension] * 2);
+
+
   } else {
     options.srcSetBreakpoints.forEach(breakpoint => {
       if (breakpoint < 1) {
